backend-diagram-converter/core/src/test/java/org/camunda/community/migration/converter/ExpressionTransformerTest.java

```diff
@@ -64,6 +66,29 @@ public Stream<DynamicTest> shouldResolveExpression() {
         this::testExpression);
   }
 
+  @Test
```

Comment:
Could we add these tests to the TestFactory above? The `ExpressionTestDataSet` could be extended by `expectExecutionUsed` and `expectMethodInvocation` or something similar and asserted as well.

This would improve scalability of our expression testing.

Reply:
I think CSV parameterized tests are sufficient. Am extended factory would be overengineered.

Reply:
Actually, it would be less code. Would you mind if I change it?